lib/src/helpers.dart

```diff
@@ -148,3 +150,61 @@ num convertArea(num area,
 
   return (area / startFactor) * finalFactor;
 }
+
+/// Creates a {@link Point} {@link Feature} from a Position.
+///
+/// @name point
+/// @param {Array<number>} coordinates longitude, latitude position (each in decimal degrees)
+/// @param {Object} [properties={}] an Object of key-value pairs to add as properties
+/// @param {Object} [options={}] Optional Parameters
+/// @param {Array<number>} [options.bbox] Bounding Box Array [west, south, east, north] associated with the Feature
+/// @param {string|number} [options.id] Identifier associated with the Feature
+/// @returns {Feature<Point>} a Point feature
+/// @example
+/// var point = turf.point([-75.343, 39.984]);
+///
+/// //=point
+Feature<Point> point(Position? coordinates, Map<String, dynamic> properties, {Map<String, dynamic>? options}) {
+  assert(coordinates != null);
+
+  final point = Point(coordinates: coordinates!);
+
+  return feature(point, properties, options: options);
+}
+
+/// Wraps a GeoJSON {@link Geometry} in a GeoJSON {@link Feature}.
+///
+/// @name feature
+/// @param {Geometry} geometry input geometry
+/// @param {Object} [properties={}] an Object of key-value pairs to add as properties
+/// @param {Object} [options={}] Optional Parameters
+/// @param {Array<number>} [options.bbox] Bounding Box Array [west, south, east, north] associated with the Feature
+/// @param {string|number} [options.id] Identifier associated with the Feature
+/// @returns {Feature} a GeoJSON Feature
+/// @example
+/// var geometry = {
+///   "type": "Point",
+///   "coordinates": [110, 50]
+/// };
+///
+/// var feature = turf.feature(geometry);
+///
+/// //=feature
+Feature<G> feature<G extends GeometryObject, P>(G geom, P? properties, {Map<String, dynamic>? options}) {
```

Comment:
same here, I don't these helper functions are necessary in Dart, because we have proper OOP with nice constructors